nixos/modules/services/network-filesystems/juicefs.nix

```
@@ -0,0 +1,200 @@
+{
+  config,
+  lib,
+  pkgs,
+  ...
+}:
+
+let
+  cfg = config.services.juicefs;
+
+  volumeOptions = lib.types.submodule {
+    options = {
+      # Basic options
+      name = lib.mkOption {
+        type = lib.types.str;
+        description = "Name of the JuiceFS volume";
+      };
```

```suggestion
  volumeOptions = lib.types.submodule ( {name, ...}: {
    options = {
      # Basic options
      name = lib.mkOption {
        type = lib.types.str;
        default = name;
        description = "Name of the JuiceFS volume";
      };
```

This way we don't need to specify the name twice.